lib/config/account/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = accountConfig
+
+var defaultsDeep = require('lodash').defaultsDeep
+
+function accountConfig (state, callback) {
+  var usersDb = new state.config.PouchDB(state.db.authenticationDb)
+  usersDb.installUsersBehavior()
```

Comment:
I think that while the logic that depends on this plugin exists in `@hoodie/server`, we should probably have `pouchdb-users` as a dependency - otherwise there'll be quite a spread of code between `hoodie` and `hoodie/server`?

Makes sense that the server relies on a PouchDB constructor, too - means they have the control over the instance etc.